### PR TITLE
fail2ban: update to 1.0.2

### PR DESCRIPTION
**Pull request by MeganerdNL**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - armv6l-glibc

#### Why this was necessary
There are many changes in fail2ban recently, especially for Python 3.11 - which is standard Python 3 in Void Linux now. Solves some compile errors in often used jail configs.

### CODE REVIEW

**Comment by necrophcodr** [comment]
@MeganerdNL You may want to read https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#committing-your-changes on how to format commits.

File: srcpkgs/fail2ban/files/fail2ban/run [hunk 1]
```diff
@@ -2,4 +2,4 @@
 exec 2>&1
 mkdir -p /var/run/fail2ban
 
-exec fail2ban-server -f
+exec fail2ban-server -f >/dev/null 2>&1
```
Comment (necrophcodr) [inline srcpkgs/fail2ban/files/fail2ban/run:5]: Be aware that this _may_ drop any errors or logging output happening from the server.

Comment (mhmdanas) [reply srcpkgs/fail2ban/files/fail2ban/run:5]: What's the point of this change?

Comment (MeganerdNL) [reply srcpkgs/fail2ban/files/fail2ban/run:5]: > What's the point of this change?

It prevents the spamming the local login with messages every time you change something (Server OK and so on). I can remove it so users van add it themselves if it has other unwanted effects I'm not aware of.

**Comment by MeganerdNL** [comment]
> @MeganerdNL You may want to read https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#committing-your-changes on how to format commits.

Thanks. Sorry, I did read it, but thought using git only was ok too. 
If it's a requirement I will use it from now on.

**Comment by Bnyro** [comment]
Following the proper commit message style is a requirement for the PR to become merged.
You can run `git commit --amend -m "fail2ban: update to 1.0.2.` and `git push -f` to fix the commit message.

**Comment by MeganerdNL** [comment]
- I removed ">/dev/null 2>&1" in the run command as it could indeed prevent important messages per @necrophcodr / @mhmdanas 
- I corrected the commit message style as required per @Bnyro.

**Comment by Bnyro** [comment]
Please run `git rebase -i master` instead of merging the changes.

**Comment by MeganerdNL** [comment]
> Please run `git rebase -i master` instead of merging the changes.

Yes I will in the future. Relatively new to git and github. Quick learner. Let me know if I have to change anything now - it seems not,. Cheers.

**Comment by Bnyro** [comment]
Yes, you should fix that now because your PR bow contains 2 useless commits where you merged with the latest master changes. Otherwise your PR can't be merged because it destroys the commit history.

**Comment by MeganerdNL** [comment]
> Yes, you should fix that now because your PR now contains 2 useless commits where you merged with the latest master changes. Otherwise your PR can't be merged because it destroys the commit history.

I did what you said. But it complained. Then I forced pushed it. Is it oke now? Only 1 commit... so it seems. Really sorry for the trouble. Thanks for the help.

**Comment by Bnyro** [comment]
Yes, the changes in the PR looks good to me now!
Didn't test the package though.

**Comment by MeganerdNL** [comment]
Ok. last force-push unnecessary. Nothing happened. Learned something.

**Pull request merged**